pkgs/html/lib/src/constants.dart

```diff
@@ -376,27 +376,86 @@ const specialElements = [
   (Namespaces.html, 'wbr'),
   (Namespaces.html, 'xmp'),
   (Namespaces.svg, 'foreignObject')
-];
+};
 
-const htmlIntegrationPointElements = [
+const htmlIntegrationPointElements = {
   (Namespaces.mathml, 'annotaion-xml'),
   (Namespaces.svg, 'foreignObject'),
   (Namespaces.svg, 'desc'),
   (Namespaces.svg, 'title')
-];
+};
 
-const mathmlTextIntegrationPointElements = [
+const mathmlTextIntegrationPointElements = {
   (Namespaces.mathml, 'mi'),
   (Namespaces.mathml, 'mo'),
   (Namespaces.mathml, 'mn'),
   (Namespaces.mathml, 'ms'),
   (Namespaces.mathml, 'mtext')
-];
+};
+
+abstract final class Charcode {
+  static const int kNull = 0x00;
+
+  /// '\t'
+  static const int kTab = 0x09;
+
+  /// '\n'
+  static const int kLineFeed = 0x0A;
+  static const int kFormFeed = 0x0C;
+
+  /// '\r'
+  static const int kCarriageReturn = 0x0D;
+
+  /// ' '
+  static const int kSpace = 0x20;
+
+  /// '"'
+  static const int kDoubleQuote = 0x22;
+
+  /// '&'
+  static const int kAmpersand = 0x26;
+
+  /// "'"
+  static const int kSingleQuote = 0x27;
+
+  /// '-'
+  static const int kHyphen = 0x2D;
 
-const spaceCharacters = ' \n\r\t\u000C';
+  /// 0
+  static const int kZero = 0x30;
 
-const int newLine = 10;
-const int returnCode = 13;
+  /// '<'
+  static const int kLessThan = 0x3C;
+
+  /// '='
+  static const int kEquals = 0x3D;
+
+  /// '>'
+  static const int kGreaterThan = 0x3E;
+
+  /// A
+  static const int kUpperA = 0x41;
```

Comment:
nit: We usually don't use `kFooBar` when identifying constants in Dart, instead we just use `fooBar`.

Reply:
Done